ui/models/taskcluster.js

```diff
@@ -28,7 +28,7 @@ export default class TaskclusterModel {
       taskId: taskId || null,
       input,
     }, staticActionVariables);
-    const queue = new Queue({ credentialAgent: taskcluster.getAgent() });
+    const queue = taskcluster.getQueue();
```

Comment:
You'll run into the same problem on lines 48 and 49 with not having the root URL in the constructor options.